pipelines/mix/tasks/get_mixed.py

```diff
@@ -0,0 +1,164 @@
+            { 'name': self.input_artifact_residential_components, 'read_type': SupportedFileReadType.DATA },
+            { 'name': self.input_artifact_buildingtype_dict, 'read_type': SupportedFileReadType.CONFIG }
+        ] 
+        self.task_function = self._task
+
+    def _get_data(self):
+        return self.load_data(self.my_data_files)
+        
+    def _task(self):
+        data_map = self._get_data()
+        
+        self.commercial_enduses = data_map[self.input_artifact_commercial_enduses]
+        self.residential_enduses = data_map[self.input_artifact_residential_enduse]
+
+        self.commercial_components = data_map[self.input_artifact_commercial_components]
+        self.residential_components = data_map[self.input_artifact_residential_components]
+
+        self.components = ['MotorA', 'MotorB', 'MotorC', 'MotorD', 'PE', 'Stat_P_Res', 'Stat_P_Cur']
+
+        self.buildingtype_dict = data_map[self.input_artifact_buildingtype_dict]
+
+        self.residential_mix = data_map[self.input_artifact_residential_mix]
+        self.residential_mix = self.residential_mix.set_index(self.residential_mix.columns[0])
+
+        residential_mix_output = self.get_mixed_output(self.residential_mix)
+        residential_mix_output.insert(loc=1, column='mix', value='residential')
+
+        self.commercial_mix = data_map[self.input_artifact_commercial_mix]
+        self.commercial_mix = self.commercial_mix.set_index(self.commercial_mix.columns[0])
+
+        commercial_mix_output = self.get_mixed_output(self.commercial_mix)
+        commercial_mix_output.insert(loc=1, column='mix', value='commercial')
+
+        self.mixed_mix = data_map[self.input_artifact_mixed_mix]
+        self.mixed_mix = self.mixed_mix.set_index(self.mixed_mix.columns[0])
+
+        mixed_mix_output = self.get_mixed_output(self.mixed_mix)
+        mixed_mix_output.insert(loc=1, column='mix', value='mixed')
+
+        self.rural_mix = data_map[self.input_artifact_rural_mix]
+        self.rural_mix = self.rural_mix.set_index(self.rural_mix.columns[0])
+
+        rural_mix_output = self.get_mixed_output(self.rural_mix)
+        rural_mix_output.insert(loc=1, column='mix', value='rural')
+
+        self.validate(residential_mix_output)
+        self.validate(commercial_mix_output)
+        self.validate(mixed_mix_output)
+        self.validate(rural_mix_output)
+        self.on_complete({self.output_artifact_residential_mix: residential_mix_output, self.output_artifact_commercial_mix: commercial_mix_output,
+                        self.output_artifact_mixed_mix: mixed_mix_output, self.output_artifact_rural_mix: rural_mix_output})           
+
+    def get_mixed_output(self, mix_chart): 
+        """This function adapt the customer_chart
+        """
+
+        output_df = pd.DataFrame(columns=['target', 'daytype', 'time'] + self.components)
+        
+        for location in self.residential_components.target.unique():
+            
+            initialization = False
```

Comment:
There's examples in the RBSA pipeline on how to bypass the need for this flag. Check it out and update this flow.

Reply:
Are you sure, I couldn't find it?

Reply:
This is the commit: https://github.com/slacgismo/loadinsight/commit/7dbbdec84b5ea2dfb9a756f0e514ec9cb258fb07